home/tests/import-export-data/list_items_with_comma_output.csv

```diff
@@ -0,0 +1,6 @@
+structure,message,page_id,slug,parent,web_title,web_subtitle,web_body,whatsapp_title,whatsapp_body,whatsapp_template_name,whatsapp_template_category,example_values,variation_title,variation_body,list_title,list_items,sms_title,sms_body,ussd_title,ussd_body,messenger_title,messenger_body,viber_title,viber_body,translation_tag,tags,quick_replies,triggers,locale,next_prompt,buttons,image_link,doc_link,media_link,related_pages,footer
+Menu 1,0,12,index,,Index,,,,,,,,,,,,,,,,,,,,573ff6a0-ed9c-4fe9-8285-7990ee12675f,,,,English,,,,,,,
+Sub 1.1,1,4,test-1,Index,Test 1,,,Test messages,texting 123,,UTILITY,"EV 2, EV 1",,,List Title,"[{""type"": ""next_message"", ""title"": ""item 1""}, {""type"": ""next_message"", ""title"": ""item 2""}]",,,,,,,,,17f43f94-a78e-4a63-86db-a32f2d70248c,,,,English,NP 1,"[{""type"": ""next_message"", ""title"": ""Btn 2""}, {""type"": ""next_message"", ""title"": ""Btn 1""}]",,,,,
+Sub 1.1.1,1,6,list-item-test,Test 1,List item test,,,List item test,message test,,UTILITY,,,,List Title,"[{""type"": ""next_message"", ""title"": ""Item 3""}, {""type"": ""next_message"", ""title"": ""Item 4""}, {""type"": ""next_message"", ""title"": ""Item 2""}, {""type"": ""next_message"", ""title"": ""Item 1""}]",,,,,,,,,f351c9f0-751c-466e-acb6-09d607410491,,,,English,,"[{""type"": ""next_message"", ""title"": ""Btn 1""}, {""type"": ""next_message"", ""title"": ""Btn 2""}]",,,,,
+Sub 1.1.1.1,1,10,importexport,List item test,import/export Test,,,import/export test,import/export test,,UTILITY,,,,List Title,"[{""type"": ""next_message"", ""title"": ""item 2, with comma""}, {""type"": ""next_message"", ""title"": ""item 3""}, {""type"": ""next_message"", ""title"": ""item 1""}, {""type"": ""next_message"", ""title"": ""Item' comma""}]",,,,,,,,,cd9f38ec-6cd9-4289-873a-417d996493b4,,,,English,,[],,,,,
+Sub 1.1.1.2,1,11,test-import,List item test,Test import,,,Test import,Test import,,UTILITY,,,,List Title,"[{""type"": ""next_message"", ""title"": ""item1""}, {""type"": ""next_message"", ""title"": ""item2""}]",,,,,,,,,6387db5b-ed76-4d34-86f4-92fbd9e1ebda,,,,English,,[],,,,,
```

Comment:
I think we should have at least one test that uses the `go_to_page` list item type.

Reply:
ha, and adding that test revealed that I hadn't fully implemented the logic for that!